tests/utils/lora/test_merge_lora.py

```
@@ -2031,6 +2031,431 @@ def fused_dequant(keys, layer, projs):
+
+        from axolotl.cli.utils.lora_merge import (
+            _build_peft_layer_and_get_delta,
+            _find_param_wrapper_lora,
+            _Nvfp4ExpertMergeWriter,
+        )
+        from axolotl.integrations.kernels.libs.sonicmoe.nvfp4_quant import (
+            fake_quant_nvfp4,
+        )
+
+        torch.manual_seed(3)
+        E, H, I, r, alpha = 2, 64, 32, 4, 8
```

Ruff (0.15.20)

[error] Ambiguous variable name: `I`

(E741)